tests/models/test_changestar.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import itertools
+
+import pytest
+import torch
+from torch.nn.modules import (
```

Comment:
Shouldn't these be imported directly from `torch.nn`? Also, if you `import torch.nn as nn` instead, it would greatly reduce the number of lines of code.

Reply:
Directly using `torch.nn as nn` will raise mypy issues.

Reply:
We do it in other files, see `torchgeo/models/fcn.py`